src/TextAnalysis.jl

```diff
@@ -66,7 +66,7 @@ module TextAnalysis
 
     export CRF, viterbi_decode, crf_loss
 
-    export NERTagger, Tracker, Flux
+    export NERTagger, POS_Tagger, Tracker, Flux
```

Comment:
Sentence Case and underscore don't go together. It should be `PoSTagger` or `pos_tagger`. For a type, probably the former. Certainly not `POS_Tagger`

Reply:
Changed this. Also changed `POS_Char_UNK` and `POS_Word_UNK` to `PoSCharUNK` and `PoSWordUNK` respectively.